packages/app/src/app/pages/Pro/Upgrade.tsx

```diff
@@ -260,13 +260,22 @@ export const ProUpgrade = () => {
                 isHighlighted
               >
                 <Stack gap={1} direction="vertical">
-                  <Text aria-hidden size={32} weight="500">
-                    $12
+                  <Text size={32} weight="500">
+                    {formatCurrency({
+                      currency: 'USD',
+                      amount: pro?.prices?.individual.year.usd / 12,
```

Comment:
`formatCurrency` expects `amount` to be a number, but if `pro?.prices?.individual.year.usd` evaluates to `undefined`, it'll receive `NaN` and possibly error.

```suggestion
                      amount: (pro?.prices?.individual.year.usd || 0) / 12,
```

Reply:
Ah yes I should fix this. We don't want to show $0 though I think, so I'm making undefined an option and return null instead. It shouldn't ever be falsy unless it's still loading the prices from somewhere.